.github/workflows/changesets.yml

```diff
@@ -21,19 +20,31 @@ jobs:
               with:
                   fetch-depth: 0
 
+            - uses: pnpm/action-setup@v4
+              with:
+                  version: 9
```

Comment:
If you just add `"packageManager": "pnpm@9.3.0",` in the root `package.json` then you don't have to add `with: version: 9` so many places

Reply:
oh, neat! didn't know that was a feature of the action. thanks for the suggestion